src/components/Modals/OnchainIdentityModal.js

```diff
@@ -54,6 +65,9 @@ function OnchainIdentityModal({
 
     isKnownGood = parseCitizenshipJudgement(judgements);
   }
+  const isWarning = identity.isSome && isKnownGood;
+
+  const [isUserAllowChangeIdentity, setIsUserAllowChangeIdentity] = useState(!isWarning);
```

Comment:
User is always allowed to change identity.

Did you mean to name it isUserWARNChangeIdentity?